### DISCUSSION TITLE
Work completed

### DISCUSSION THREAD

**Discussion by Yakuza112v2** [Courseplay/Courseplay_FS22#3102] [General]
Is it possible that the work finished is set so that it is only displayed to the owner of the tractor in multiplayer and not to everyone who is on the server?

**Comment by Tensuko** [comment]
Not sure if that could be done per Farm ID, would have to check that.
But that still means everyone who is in the same farm.
There is no other way.

**Reply by Yakuza112v2** [reply]
That would be great if it worked like this via FarmID.

**Comment by Yakuza112v2** [comment]
Could you tell me where I have to change the FarmID?

**Reply by Tensuko** [reply]
Cant do yet
